src/Swashbuckle.AspNetCore.SwaggerGen/SwaggerGenerator/SwaggerGeneratorOptions.cs

```diff
@@ -6,7 +6,6 @@
 #endif
 using Microsoft.AspNetCore.Mvc.ApiExplorer;
 using Microsoft.OpenApi.Models;
-using Microsoft.AspNetCore.Routing;
```

Comment:
Missed this - needs putting back as it breaks the build.

Reply:
Shouldn't it be in the compiler switch? Because VS is saying it is not used

Reply:
If only some TFMs use it, then you'll need to wrap it in an `#if` to stop the suggestion to remove it.

Reply:
Yeah, it is needed, still strange that visual studio isn't recognizing it as being used

Reply:
restored it

Reply:
It's probably because you have a TFM that isn't the one that needs it selected as the current "view" (I don't know what the terminology for it is) you have of the code open in the document well.